### DISCUSSION TITLE
[Feature] - One shortcut for each direction

### DISCUSSION THREAD

**Discussion by dinbtechit** [rxhanson/Rectangle#974] [Ideas]
Instead of 30 shortcuts, how about one shortcut and cycle through the different sizes? Similar to how it is in Linux and windows.

<kbd>Command</kbd> + ➡  -  Right-half -> 3/4 of Right-half -> Right-Half 
<kbd>Command</kbd> +⬅ -   Left-half -> 3/4 of Left-half -> Left-Half 
<kbd>Command</kbd> + ⬆ -  (only when minimized) -> initial -> 1/2 top half ->  maximum size ->  full screen -> initial window size
<kbd>Command</kbd> + ⬇ -  initial -> 1/2 bottom half ->  minimize

**Comment by rxhanson** [comment]
Thanks for the feedback. I don't have any plans to add something like this into Rectangle, but see no reason to turn down a good pull request for it. At first cut, it would have to be a Terminal command enabled flag and not in the UI.

Alternatively, you could use [Rectangle Pro](https://rectangleapp.com/pro) (paid). It has:

1. A "Win arrow keys" option in the Keyboard Shortcuts tab where you can use modifier+arrow keys to follow a state machine pattern like Windows.
2. The ability to create custom shortcuts, and specify what happens on each repeated key press of that shortcut.
3. The Window Throw capability that gets you 16 different sizes/positions with just one modifier combo and cursor movement.
